app/components/termsoverlay/termsoverlay.js

```diff
@@ -97,9 +232,9 @@ var TermsOverlay = React.createClass({
 
     var submit = this.props.onSubmit;
     if (submit) {
```

Comment:
Related to my question above about requiring the `onSubmit` prop...why would we ever expect it to be absent??